vnet/lib/vnet/endpoints/1.0/vnet_api.rb

```diff
@@ -115,6 +115,21 @@ def update_by_uuid(class_name, fill = {})
       respond_with(response.generate(updated_object))
     end
 
+    def update_by_uuid2(class_name, fill = {})
```

Comment:
The old update/update_uuid refactoring in node_api isn't yet done.

When a class node_api is refactored then the above method needs to be used or the unit tests fail so logging doesn't really change anything.